.github/workflows/ui-test-users-id-tasks-id.yml

```diff
@@ -0,0 +1,52 @@
+name: UI Test /users/{userid}/tasks/{taskid}
+
+on: 
+  push:
+    branches:
+      - development
+      - natalia-uitest-sprint4
+  pull_request:
+    branches: 
+      - development
+      - natalia-uitest-sprint4
+
+jobs:
+
+  ui-test-taskid:
+    runs-on: ubuntu-latest
+    
+    env:
+      NODE_VERSION: '14'
+      UI_TESTS_PATH: './test/ui'
+      UI_RESULTS_TASKID_FILE_PATH: './test/results/ui-test-results/users-id-tasks-id/'
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Use Node.js ${{ env.NODE_VERSION }}
+        uses: actions/setup-node@v2
+        with:
+          node-version: ${{ env.NODE_VERSION }}
+          check-latest: true
+
+      - name: Run the ui test-taskid
+        run: |
+          pushd ${{ env.UI_TESTS_PATH }}
+          npm install
+          npm run ui-test-taskid
+          popd
+
+      - name: Upload screenshots to Azure Blob Storage
+        uses: azure/CLI@v1
+        with:
+          inlineScript: |
+            az storage blob upload-batch \
+            --destination screenshots \
+            --source ${{ env.UI_RESULTS_TASKID_FILE_PATH }} \
+            --connection-string "${{ secrets.AZBLOB_CONNECTION_STRING }}" \
+            --pattern "*.png"
+
+      - name: Azure logout
```

Comment:
I don't think you ever logged in so this is probably unnecessary

Reply:
removed